integration/nwo/standard_networks.go

```diff
@@ -183,6 +183,107 @@ func BasicEtcdRaft() *Config {
 	return config
 }
 
+func BasicEtcdRaftNoSysChan() *Config {
```

Comment:
can't we simply use other functions like BasicEtcdRaft() and then just nil out the system channel?  Same for the multi channel function below.

Reply:
We're using `BasicConfig()` as the starting point.

Once we're done with removing the system channel, `BasicEtcdRaft` will become unused (and will be eventually removed), and replaced everywhere by `BasicEtcdRaftNoSysChan`. Same goes for all other standard networks.

This is just for the transition period when we have both, this way it is easy to detect which tests where converted and which are still using the system channel.

Reply:
I understand the intent. 

What I'm trying to understand is, why do you need to create a no-system channel version of any network config constructor, instead of calling the constructor and then just overriding selectively the bootstrap method, channel participation, and system channel?

For example, in `MultiChannelEtcdRaftNoSysChan()` you have:

```
config.Consensus.Type = "etcdraft"
	config.Profiles = []*Profile{
		{
			Name:          "TwoOrgsAppChannelEtcdRaft",
			Consortium:    "SampleConsortium",
			Orderers:      []string{"orderer"},
			Organizations: []string{"Org1", "Org2"},
		},
	}
	config.Channels = []*Channel{
		{Name: "testchannel", Profile: "TwoOrgsAppChannelEtcdRaft"},
		{Name: "testchannel2", Profile: "TwoOrgsAppChannelEtcdRaft"},
	}

	for _, peer := range config.Peers {
		peer.Channels = []*PeerChannel{
			{Name: "testchannel", Anchor: true},
			{Name: "testchannel2", Anchor: true},
		}
	}
```

why?  Is it not possible to call `MultiChannelEtcdRaft` and override only what's needed? 
In general, for a `Foo()` is it not possible to simply create a `FooNoSysChannel()` that calls `Foo()` and then just overrides system channel, bootstrap method and channel participation? This way it's more clear and the cleanup is also more clear - just fold the two functions into one.